libs/dh/message-archive/feature-log-search/src/lib/searchresult/dh-message-archive-log-search-result.component.html

```diff
@@ -36,14 +36,24 @@
     <!-- sender gln -->
     <ng-container matColumnDef="sender">
       <th mat-header-cell *matHeaderCellDef>{{ transloco("sender") }}</th>
-      <td mat-cell *matCellDef="let element">{{ element.senderGln }}</td>
+      <td mat-cell *matCellDef="let element">
+        <container-element [ngSwitch]="element.httpData">
+          <label *ngSwitchCase="'request'" type="info">{{
+            element.senderGln
+          }}</label>
+          <label *ngSwitchCase="'response'" type="success">{{
+            element.receiverGln
+          }}</label>
+          <label *ngSwitchDefault type="info">{{ "??" }}</label>
```

Comment:
nitpick: the interpolation (`{{ .. }}`) is not necessary when we want to output a simple text string.

I.e. `<element>??</element>`